packages/language-php/grammars/tree-sitter/queries/highlights.scm

```diff
@@ -473,7 +509,15 @@
   "-="
   "*="
   "/="
+  "%="
+  "**="
+  "&="
+  "|="
+  "^="
+  "<<="
+  ">>="
```

Comment:
These are all defined at https://www.php.net/manual/en/language.operators.assignment.php#language.operators.assignment.arithmetic

Reply:
`??=` implies the existence of `??`, which we didn't support yet. In the course of discovering `??` I also found `?:`. I'll add both to the logical operators list.

Reply:
Leaving `?:` off because the PHP parser doesn't recognize it as one unit. But it does end up looking right because the `?` and the `:` are individually scoped as ternary operators, so we've caught a break.

Reply:
Makes sense. I had tried `?:` and it seemed to be working fine, so I didn't look further. `??` is a good catch, though.